src/main.ts

```diff
@@ -131,14 +141,19 @@ export class Purchases {
     }
     validateApiKey(apiKey);
     validateAppUserId(appUserId);
-    Purchases.instance = new Purchases(apiKey, appUserId);
+    Purchases.instance = new Purchases(apiKey, appUserId, httpConfig);
     return Purchases.getSharedInstance();
   }
 
   /** @internal */
-  private constructor(apiKey: string, appUserId: string) {
+  private constructor(
+    apiKey: string,
+    appUserId: string,
+    httpConfig?: HttpConfig,
+  ) {
     this._API_KEY = apiKey;
     this._appUserId = appUserId;
+    this._httpConfig = httpConfig ? httpConfig : { includeCredentials: false };
```

Comment:
Nitpick but we could also leave the `_httpConfig` to null if it's not passed in? NABD though.

Reply:
I added a default value to avoid this ternary expression.

Reply:
Ok that's fine too 👍